app/templates/codelijsten/edit.hbs

```diff
@@ -49,161 +51,174 @@
         >
           <AuButton
             @skin="link"
-            @icon="export"
+            @icon="archive"
             @iconAlignment="left"
             role="menuitem"
             @disabled={{this.isPrivateConceptScheme}}
-            @loadingMessage={{t "messages.loading.downloading"}}
-            {{on "click" this.exportCodelist}}
+            @loadingMessage={{t "messages.loading.isArchiving"}}
+            {{on "click" (fn (mut this.isArchiveModalOpen) true)}}
           >
-            {{t "actions.exportCodelist"}}
+            {{t "crud.archive"}}
           </AuButton>
           <AuButton
             @skin="link"
-            @icon="bin"
+            @icon="export"
             @iconAlignment="left"
-            @alert={{true}}
             role="menuitem"
             @disabled={{this.isPrivateConceptScheme}}
-            @loadingMessage={{t "messages.loading.isDeleting"}}
-            {{on "click" (fn (mut this.isDeleteModalOpen) true)}}
+            @loadingMessage={{t "messages.loading.downloading"}}
+            {{on "click" this.exportCodelist}}
           >
-            {{t "crud.delete"}}
+            {{t "actions.exportCodelist"}}
           </AuButton>
         </AuDropdown>
       {{/if}}
     </div>
   </Group>
 </AuToolbar>
-
-<div class="codelijsten__scroll">
-  <span class="au-u-padding">
-    {{#if this.isPrivateConceptScheme}}
-      <AuAlert @title={{"Opgelet"}} @skin={{"warning"}} @icon={{"login"}}>
-        <p>{{t "codelists.alert.genericCodelist"}}</p>
-      </AuAlert>
-    {{/if}}
-    <div class="au-u-3-5 au-u-margin-bottom-large">
-      <AuLabel for="input-codelist-name" @required={{true}}>
-        {{t "codelists.nameOfCodelist"}}
-      </AuLabel>
-      <AuInput
-        id="input-codelist-name"
-        @width="block"
-        @error={{this.nameErrorMessage}}
-        @disabled={{this.isPrivateConceptScheme}}
-        value={{this.codelistName}}
-        {{on "blur" this.handleNameChange}}
-      />
-      {{#if this.nameErrorMessage}}
-        <AuHelpText @error={{true}}>{{this.nameErrorMessage}}</AuHelpText>
-      {{/if}}
-    </div>
-    <div class="au-u-3-5 au-u-margin-bottom-large">
+{{#if this.setup.isRunning}}
+  <AuLoader />
+{{else}}
```

Comment:
Add a loader to the content so it is not flickering between previous states of the codelist detail